feature-group-definitions/map.yml

```diff
@@ -0,0 +1,49 @@
+name: Map (initial support)
+description: Map objects hold key-value pairs and remember the original insertion order of the keys.
+spec:
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map-objects
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map-constructor
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.clear
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.delete
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.entries
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.foreach
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.get
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.has
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.keys
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.set
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-get-map.prototype.size
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype.values
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-map.prototype-@@iterator
+  - https://tc39.es/ecma262/multipage/keyed-collections.html#sec-get-map-@@species
+snapshot: ecmascript-2015
+group: maps
+status:
+  baseline: high
+  baseline_low_date: 2016-09-20
+  support:
+    chrome: "51"
```

Comment:
This doesn't match the initial support of the feature, which is Chrome 38 in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#browser_compatibility

For initial support features, I think we should stick to the earliest release dates, and either exclude BCD keys or use editorial override when those things aren't very important.

Reply:
The outlier here is `@@species` which was part of the initial ecmascript-2015 Maps snapshot but shipped later because the ecmascript-2015 snapshot was so large.

I don't think it `@@species` is that important to split it out to a separate bundle. I think it belongs to this bundle with all the other ecmascript-2015 Map features.

I guess that leaves me with an editorial override. How do I do that?

Reply:
ah, Chromestatus did make this a feature https://chromestatus.com/feature/4750327530127360 (entry created on 2016-03-16). Not sure if we would create a species bundle 8 years later still. (I can, though)

Reply:
An editorial override is just a comment that will keep us from changing the data when we generate statuses from BCD. Current cases:

https://github.com/web-platform-dx/web-features/blob/b0932750cfd91a908fe6248c72c5fd66a9a835bf/feature-group-definitions/lab.yml#L7-L12

https://github.com/web-platform-dx/web-features/blob/b0932750cfd91a908fe6248c72c5fd66a9a835bf/feature-group-definitions/js-modules.yml#L14-L15

https://github.com/web-platform-dx/web-features/blob/b0932750cfd91a908fe6248c72c5fd66a9a835bf/feature-group-definitions/trusted-types.yml#L8-L9

https://github.com/web-platform-dx/web-features/blob/b0932750cfd91a908fe6248c72c5fd66a9a835bf/feature-group-definitions/container-queries.yml#L9-L11

https://github.com/web-platform-dx/web-features/blob/b0932750cfd91a908fe6248c72c5fd66a9a835bf/feature-group-definitions/oklab.yml#L7-L12

Reply:
Until we have a structure for it, we ought to use a conventional prefix for override comments. `OVERRIDE`, maybe?

Reply:
Looking at the examples of current editorial overrides, I wonder why we're are adjusting the status block manually.

I guess I would add a list of features to ignore for the status block and not tweak the `compat_features` list and neither update version numbers manually.

```yaml
compat_features_ignored_for_status: 
# @@species shipped later but is not important for using initial Map objects
- javascript.builtins.Map.@@species
```

The `status` block would take into account both lists and we can produce the baseline status and browser versions programmatically.

Reply:
Added an editorial override as a comment but I think it would be nice to have a structure in the data itself. Ignore list s come to mind but maybe we want more editorial overrides as comments until we see a good structure for it.